go/libkb/sig_chain.go

```diff
@@ -408,6 +420,17 @@ func (sc *SigChain) Store(ctx context.Context) (err error) {
 	return nil
 }
 
+// Clean will remove any unnecessary memory consumed by
```

Comment:
See comment above, there's a bug here w/r/t resets.

Reply:
Where is a test for it?  The reset user tests are passing.

We're not dropping the link, just the unpacked sig.

Reply:
Yeah, I meant the unpacked sig. I'd guess the tests are passing because we don't have a sigchain of the form:

1: eldest
2: track
3: eldest

Reply:
The links are verified before the sig is dropped, so they get the `payloadVerified` and `hashVerified` flags set to true.  So any future verifications will pass with those links.